### PR TITLE
generate docs: disable md auto links

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
When generating docs disable auto linking to human readable output http links. This output has a lot of times garbage links and is disabled by default in the Server. This change makes human readable output act similar regarding linking as is in the Server.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by glicht** [comment]
@roysagi added the following line to pytest.ini:
```
testpaths = demisto_sdk
```
To avoid pytest searchinig for tests outside of the `demisto_sdk` dir.

File: demisto_sdk/commands/generate_docs/tests/generate_docs_test.py [hunk 1]
```diff
@@ -939,3 +940,16 @@ def test_generate_versions_differences_section(monkeypatch):
     ]
 
     assert section == expected_section
+
+
+def test_disable_md_autolinks():
```
Comment (roysagi) [inline demisto_sdk/commands/generate_docs/tests/generate_docs_test.py:945]: Please add a given when then docstring

Comment (glicht) [reply demisto_sdk/commands/generate_docs/tests/generate_docs_test.py:945]: added

**Comment by glicht** [comment]
@roysagi waiting for you. docstring was fixed...

**Review by roysagi — APPROVED** [review]

**Pull request merged**